extensions/amp-gist/amp-gist.md

```diff
@@ -47,13 +47,30 @@ Example:
 </amp-gist>
 ```
 
+### Single file
+
+```html
+<amp-gist
+    data-gistid="a19e811dcd7df10c4da0931641538497"
+    data-file="hi.c"
+    layout="responsive"
+    width="480" height="270">
+</amp-gist>
+```
+
 ## Behavior
 
 This extension creates an iframe and displays the gist from GitHub.
 
 ## Attributes
 
-It requires the `data-gistid` attribute of the gist.
+These are the valid attributes for the `amp-gist` component:
+
+**data-gistid** (required)
+The ID of the gist to embed.
+
+**data-file** (optional)
```

Comment:
Just as an aside, all `data-` attributes are allowed by the validator. It's only when one wants to make them mandatory or put specific requirements on their values do they need to show up in the protoascii.